lib/modules/platform/scmm/readme.md

```diff
@@ -0,0 +1,13 @@
+# SCM-Manager
+
+Renovate supports [SCM-Manager](https://scm-manager.org).
+
+## Authentication
+
+First, create an API Key for your technical Renovate user in SCM-Manager.
+The technical user should be configured properly with name and email address.
+Then let Renovate use your API Key by setting the `RENOVATE_TOKEN` environment variable with your key.
+
+You must set `platform=scmm` in your Renovate config file.
```

Comment:
```suggestion
You must set the [`platform`](https://docs.renovatebot.com/self-hosted-configuration/#platform) to `scmm` in your your Renovate config file:
```

How about something like this, where we also link to the `platform` config option docs for Self-hosted users.